types/params.go

```diff
@@ -207,15 +207,16 @@ func (params ConsensusParams) ValidateBasic() error {
 
 // ValidateUpdate validates the updated VoteExtensionsEnableHeight.
 // | r | params...EnableHeight | updated...EnableHeight | result (nil == pass)
-// | 1 | *                     | (nil)                  | nil
-// | 2 | *                     | < 0                    | VoteExtensionsEnableHeight must be positive
-// | 3 | <=0                   | 0                      | nil
-// | 4 | > 0; <=height         | 0                      | vote extensions cannot be disabled once enabled
-// | 5 | > 0; > height         | 0                      | nil (disable a previous proposal)
-// | 6 | *                     | <=height               | vote extensions cannot be updated to a past height
-// | 7 | <=0                   | > height (*)           | nil
-// | 8 | (> 0) <=height        | > height (*)           | vote extensions cannot be modified once enabled
-// | 9 | (> 0) > height        | > height (*)           | nil
+// |  1 | *                    | (nil)                  | nil
+// |  2 | *                    | < 0                    | VoteExtensionsEnableHeight must be positive
+// |  3 | <=0                  | 0                      | nil
+// |  4 | X                    | X                      | nil
```

Comment:
```suggestion
// |  4 | X                    | X (>=0)                | nil
```

If you don't mind, I'd like to clarify that the rule has the implicit knowledge that `updated...EnableHeight` should be positive. (Derived from the previous rules.)

Since not all rules were processed yet where `updated...EnableHeight` can be zero, it's best to claim it's `>=0`.

Reply:
I would like to keep it as is, otherwise we risk breaking the existing behavior in another way: ">=0" is a restriction that didn't exist in `v0.38.2`. If you're keen on adding it, we can leave it on `main` but not backport it to `v0.38.x`